docs/testplan/capacity_test_2TierNetwork.md

```diff
@@ -0,0 +1,31 @@
+# Test Objective
+This test aims to track switch power usage, temperature, queue watermark and health state.
+
+# Test Setup
+![Test Setup](./2TierNetwork.png)
+
+1.	The testbed consists of four IXIA traffic generators (synchronized using a time-sync metronome) and five SONiC switches, where the BT1 switch is the Device Under Test (DUT).
+2.	Each of the four BT0 switches is connected to the DUT via eight DAC cables. There are no direct connections between any two BT0 switches.
+3.	Each BT0 switch is also connected to one IXIA traffic generator via eight optical cables. Similarly, there are no direct connections between any two IXIA devices.
+4.	Both switches and IXIAs support four port breakout modes: 8x100Gbps, 4x200Gbps, 2x400Gbps, and 1x800Gbps. However, they must operate in the same mode. In 8x100Gbps mode, each cable supports eight links. In 4x200Gbps mode, each cable supports four links. So on and so forth.
+5.	The routing configuration of the BT0 switches should ensure that all data traffic go through the DUT.
+
+# Configuration
+1. The test duration is configurable with default value of 1 hour. 
```

Comment:
same minor issue for this list and heading. better fix the format according to markdown's basic practice.

Reply:
e.g.: Only 1 level-1 heading; list should have an empty line before it; the empty line between the list item is unnecessary.